p2p/src/connection/outgoing/p2p_connection_outgoing_actions.rs

```diff
@@ -14,9 +14,6 @@ pub type P2pConnectionOutgoingActionWithMetaRef<'a> =
 #[derive(Serialize, Deserialize, Debug, Clone, ActionEvent)]
 #[action_event(fields(display(opts), display(peer_id), display(error)))]
 pub enum P2pConnectionOutgoingAction {
-    /// Initialize connection to a random peer.
-    #[action_event(level = trace)]
-    RandomInit,
```

Comment:
@0xMimir re `RandomInit` in the effectful action, this should still be here, not removed. Then it should implement the enabling condition currently in the effectful action, and when enabled, dispatch the effectful counterpart.